docs/src/content/docs/commands/GETSET.md

```diff
@@ -13,44 +13,49 @@ GETSET key value
 
 ## Parameters
 
-- `key`: The key whose value you want to retrieve and set. This must be a string.
-- `value`: The new value to set for the specified key. This must be a string.
+| Parameter       | Description                                      | Type    | Required |
+|-----------------|--------------------------------------------------|---------|----------|
+| `key`           | The key whose value you want to retrieve and set.                   | String  | Yes      |
```

Comment:
Aren't these not aligned?

Reply:
My IDE had word-wrap toggled so i guess it added those spaces, but the render was accurate when previewed in the IDE. Attaching the preview shown on git below:
![Screenshot From 2024-10-01 23-58-24](https://github.com/user-attachments/assets/1a8d86d3-ab9e-4dfa-adf9-b3ac74277e4a)

Let me know if we still need to refactor it.